src/Kmd.Logic.Gateway.Automation.Tool/ValidateCommandHandler.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Kmd.Logic.Gateway.Automation.Models;
+using Microsoft.Rest;
+
+namespace Kmd.Logic.Gateway.Automation.Tool
+{
+    internal class ValidateCommandHandler : CommandHandlerBase
```

Comment:
Why it has to be validated independently? Publish result will will have all validation errors and success

Reply:
But what if somebody wants to test its publish yml file locally without publishing? I am 100% sure that responsible person will not create whole file structure at first shot :D

Reply:
yes we can keep this option of validation only, however publish as to validate all the validation before publish

Reply:
Yeah, sure, it is a must have. But maybe it's worth to do that in the scope of Publishing user story, instead of here.